examples/README.md

```diff
@@ -1,21 +1,19 @@
 
 ## Example Measurements
 
-### Downloading the Example Measurements
+These are example measurements for MeasurementLink 2023 Q3 or later.
 
-Example measurements are available in the `examples` directory of the `measurementlink-python` Git repository. (That's this directory.) You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.
+If you are using a previous version of MeasurementLink, download the appropriate examples:
 
-> **Warning**
-> Example measurements in the `main` branch of the Git repository may require unreleased versions of InstrumentStudio and MeasurementLink, due to new features and `.measui` format changes.
+- MeasurementLink 2023 Q1: [measurementlink-python-examples-1.0.1.zip](https://github.com/ni/measurementlink-python/releases/download/1.0.1/measurementlink-python-examples-1.0.1.zip)
```

Comment:
It looks like you retroactively created the example assets? I can do that for LabVIEW too.